### ISSUE TITLE
Add MediaInfo VideoDynamicRangeType as a custom filer

### ISSUE THREAD

**Issue by undead9786** [Sonarr/Sonarr#5106]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe

No.

### Describe the solution you'd like

Add in custom filter a tab for MediaInfo VideoDynamicRangeType.

### Describe alternatives you've considered

Tagging but that is a very manual effort.

### Anything else?

Pretty straightforward request.

**Comment by markus101** [comment]
Please elaborate on exactly where you want to see this filter. It's not clear where you expect this filter to exist nor how tagging being a manual effort impacts such a filter option.

Exactly what and where are you trying to filter episode files that have `MediaInfo VideoDynamicRangeType`?

**Comment by undead9786** [comment]
> Please elaborate on exactly where you want to see this filter. It's not clear where you expect this filter to exist nor how tagging being a manual effort impacts such a filter option.
> 
> Exactly what and where are you trying to filter episode files that have `MediaInfo VideoDynamicRangeType`?

Under filters there should be an option for MediaInfo VideoDynamicRangeType and then you add the types you want such as "DV" or "DV HDR" etc etc. If you have to add tags to specific types of ranges than that would require a lot of manual intervention just to be able to filter the type.

**Comment by markus101** [comment]
Series filters? Interactive search filters? Exactly which filters?

**Comment by undead9786** [comment]
> Series filters? Interactive search filters? Exactly which filters?
> 
> Exactly what problem are you trying to solve? You've provided a solution, but not the problem you're trying to solve.

Not sure what you mean. I just got an update from Sonarr which finally added Video Dynamic Range and I want to be able to add a custom filter to be able to either show what contains it or doesn't.

**Comment by markus101** [comment]
Right, that allows you to show that information on the series details page, but that page doesn't have filters as episodes are shown per series.

Exactly what are you trying to filter? Are you trying to filter the series list to see which series have filter that have Video Dynamic Range or something else? Maybe show a screenshot of what you're looking at because this is not a "pretty straight forward request".

**Comment by undead9786** [comment]
> Right, that allows you to show that information on the series details page, but that page doesn't have filters as episodes are shown per season.
> 
> Exactly what are you trying to filter? Are you trying to filter the series list to see which series have filter that have files with Video Dynamic Range or something else? Maybe show a screenshot of what you're looking at because this is not a "pretty straight forward request".

Sorry, thought it was more straightforward. I want it to function exactly like the filter for "Release Groups" where you choose it and then choose contains or does not contain and then you put DV HDR10 or other types.
